app/renderer/js/utils/domain-util.js

```diff
@@ -20,7 +20,7 @@ const logger = new Logger({
 
 let instance = null;
 
-const defaultIconUrl = '../renderer/img/icon.png';
+const defaultIconUrl = path.resolve(__dirname, './../../img/icon.png');
```

Comment:
I'd remove the `./` here since it's redundant and looks clear without the `./`.
```suggestion
const defaultIconUrl = path.resolve(__dirname, '../../img/icon.png');
```

Reply:
Thank you for pointing that out (^_^)
Implemented the same in commit 46315ec